pkg/interceptors/cel/cel.go

```diff
@@ -58,12 +59,8 @@ func (w *Interceptor) ExecuteTrigger(request *http.Request) (*http.Response, err
 		return nil, fmt.Errorf("error creating cel environment: %w", err)
 	}
 
-	body, err := request.GetBody()
-	if err != nil {
-		return nil, fmt.Errorf("error getting request body: %w", err)
-	}
-	defer body.Close()
-	payload, err := ioutil.ReadAll(body)
+	defer request.Body.Close()
```

Comment:
This still runs the risk of panicking if request or Body is nil. We probably want to have checks here to handle gracefully.

Example - https://play.golang.org/p/oYUtY68zw7s

Reply:
Good point; let me add some checks

Reply:
testing this is going to require us to refactor our tests a bit...gonna make this a new commit in this PR!